lib/ansible/plugins/inventory/gcp_compute.py

```diff
@@ -236,8 +236,6 @@ def _return_if_object(self, module, response):
 
         if navigate_hash(result, ['error', 'errors']):
             module.fail_json(msg=navigate_hash(result, ['error', 'errors']))
```

Comment:
this is incorrect, jail_json ONLY works for modules, this won't correctly report errors in an inventory plugin.